### PR TITLE
Feature/show tutorials

### PR DESCRIPTION
**Pull request by HumnaAleem**

### CODE REVIEW

File: consumer/src/components/Footer/HelpFooter.tsx [hunk 1]
```diff
@@ -46,65 +45,79 @@ const useStyles = makeStyles((theme) => ({
 	}
 }));
 
+const getIDFromYoutbeLink = (link: string) => {
```
Comment (raoaliahmad) [inline consumer/src/components/Footer/HelpFooter.tsx:48]: why did you remove the import from utills and added the code here?

File: consumer/src/components/Footer/HelpFooter.tsx [hunk 2]
```diff
@@ -46,65 +45,79 @@ const useStyles = makeStyles((theme) => ({
...
 // 		window.location.href = "https://web.facebook.com/groups/ilmexchangediscussionforum/"
 // 	}, 25);
 // 	window.location.href = "fb://groups/ilmexchangediscussionforum/";
 // }
-
-const getYoutubeTutorialLink = () => {
-
-	// Will add mechanism to get links based on the current page in view
-	// for now just showing this link in every page
-
```
Comment (raoaliahmad) [inline consumer/src/components/Footer/HelpFooter.tsx:60]: why would you change everything when you could have just added the link switching mechanism there?

File: consumer/src/components/Footer/HelpFooter.tsx (hunk 1 above)
Comment (HumnaAleem) [reply consumer/src/components/Footer/HelpFooter.tsx]: Opss... I didn't remove it from utils. But I have corrected it now. Kindly check it out.
